javascript/leap.js

```diff
@@ -0,0 +1,16 @@
+var isLeapYear = function (year) {
+  if (year % 400 === 0) {
+    return true;
+  }
+  if (year % 100 === 0) {
+    return false;
+  }
+  if (year % 4 === 0) {
+    return true;
+  }
```

Comment:
You could potentially shorten this and eliminate the `if/else` conditionals by returning the result of your logical expressions themselves.